components/builder-graph/src/main.rs

```diff
@@ -550,16 +524,17 @@ fn do_export(graph: &PackageGraph, filename: &str, filter: &str) {
     }
 
     for ident in latest {
-        if ident.starts_with(filter) {
+        if ident.starts_with(&filter) {
             file.write_fmt(format_args!("{}\n", ident)).unwrap();
         }
     }
 }
 
-fn do_target(graph: &mut PackageGraph, target: &str) {
-    match PackageTarget::from_str(target) {
+fn do_target(graph: &mut PackageGraph, matches: &ArgMatches) {
+    println!("ClapDTM {:?}", matches);
```

Comment:
I see a few of these `ClapX` `println!` statements.  Could we expand on what the `X` is, in this case `DTM`?  We also may want to consider moveing these to `debug!` lines before merging.